examples/script_nifti.py

```diff
@@ -0,0 +1,41 @@
+import os
+import pathlib
+import shutil
+import sys
+from shimmingtoolbox.dicom_to_nifti import dicom_to_nifti
+import tempfile
+import urllib.request as ur
+from zipfile import ZipFile
+
+
+def main():
```

Comment:
Take a look at #6; I suggest using `click` for your CLIs.

I know for someone new to python jumping into a whole new dependency might be a bit overwhelming. I wouldn't recommend it if I didn't think it would really help speed up development.